code/Controllers/CMSMain.php

```diff
@@ -784,7 +784,7 @@ public function savetreenode($request)
 
     public function CanOrganiseSitetree()
     {
-        return !Permission::check('SITETREE_REORGANISE') && !Permission::check('ADMIN') ? false : true;
+        return ( Permission::check('SITETREE_REORGANISE') || Permission::check('ADMIN') );
```

Comment:
There is no case where SITETREE_REORGANISE would be false but ADMIN would return true; You should just make it the simple case.